docs/pages/operations-guide.md

```diff
@@ -106,11 +106,12 @@ The service Scheduler's `main()` function is run like any other Java application
 
 ### Reconfiguration
 
-#### Steps handled by the DC/OS cluster
+This is the flow for reconfiguring a DC/OS service either in order to update specific configuration values, or to upgrade it to a new package version.
 
-1. The user edits the Scheduler's environment variables either via the DC/OS Dashboard's Services section or via Marathon directly (at `<dcos-url>/marathon`).
+#### Steps handled by the DC/OS cluster
 
-1. Marathon kills the current Scheduler and launches a new Scheduler with the updated environment variables.
+1. The user edits the Scheduler's environment variables either using the Scheduler CLI's `update` command or via Marathon directly (at `<dcos-url>/marathon`).
```

Comment:
the marathon UI isn't being maintained anymore. I'm pretty sure the way to do this would be to go to  `Service > <service-name> > edit` in the DC/OS UI.